crmprtd/networks/bc_hydro/normalize.py

```diff
@@ -19,12 +22,25 @@ def normalize(file_stream):
     # All of the BC Hydro data seem to be either comma delimited or a
     # mixture of a tab and other whitespace to make it fixed
     # width. This pattern encapsulates either
-    sep_pattern = re.compile(b"\\s*[\t,]+\\s*")
+    sep_pattern = re.compile("\\s*[\t,]+\\s*")
 
     # Detect if numbers are convertible
-    num_pattern = re.compile(rb"-?\d+(\.\d+)?$")
+    num_pattern = re.compile(r"-?\d+(\.\d+)?$")
+
+    variable_substitutions_path = "networks/bc_hydro/variable_substitutions.yaml"
```

Comment:
Hmm, hardcoding makes me itchy, but it seems like this is a case for it. Unless we want to merge all variable substitutions in a single global substitutions file, which is probably not worth the trouble.

Reply:
Agreed in principle, but in practice, these things change very rarely. The other hard coded substitution that we have `crmprtd/networks/wamr/station_substitutions.yaml` has never changed.